### DISCUSSION TITLE
Search also in notes text

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#8200] [General]
Hello,

I am adding tons of stuff under my notes. Like URL to the item I bought online or specific details on the things I bought. For transaction description I only add short text like "Online Amazon". When I search for things I can not find transactions that I put under notes.

Would it be possible to extend the search scope to also search for text under notes?

Thank you.

**Comment by JC5** [comment]
This is already possible, but the default search will not do it. Check out https://docs.firefly-iii.org/firefly-iii/features/search/#notes

**Reply by thehijacker** [reply]
Oh yeah. Missed this one. Using notes_contain:string I can find what is written in notes. Thank you.
